src/components/molecules/Chatbox/Chatbox.scss

```diff
@@ -51,4 +52,19 @@
   &__submit-button-icon {
     color: $white;
   }
+
+  &__emoji-picker-toggler {
+    cursor: pointer;
+  }
+
+  &__emoji-picker {
+    position: absolute;
+    bottom: 65px;
+    right: 60px;
+    z-index: 2;
```

Comment:
This should be using our `z()` `z-index` layer helper from `scss/constants`

Reply:
added chatbox-emoji-picker to constants

Reply:
Was fixed 
![image](https://user-images.githubusercontent.com/63194656/120830196-ebe63200-c566-11eb-9eb0-8c4fd5a4d938.png)